source/index.html.md

```diff
@@ -183,6 +183,29 @@ This endpoint finds  customer record by its id
 
 This endpoint loads customers by csv files
 
+>**Example body for uploading a csv file containing customer records**
+
+```json
+{
+  "entries": [
+    {
+      "customerNum": "4466",
+      "firstName": "Sarah",
+      "imported": true,
+      "lastName": "Kimber",
+      "phoneNo": "0247736845",
+      "phoneNoValid": true
+    }
+  ],
+  "id": "7359ba7e-a6c4-11ec-b909-0242ac120002"
+}
+```
+
+To format the CSV, the CSV file must have the following records, and must have a heading matching the following.
+First name;Last name;Phone number;Customer number
```

Comment:
This sentence should be done the exact way i sent it to you.

```
The CSV file must have the following records, and must have a heading matching the following.
```
Followed by the following in caps on a separate line.

**First name;Last name;Phone number;Customer number**

Reply:
Beneath it, add this

* `Customer number` is not required and can be empty.